core/src/main/resources/templates/email/buddy-invitation-body_nl.vm

```diff
@@ -0,0 +1,19 @@
+<html>
```

Comment:
Maybe add lang="nl"? (See https://www.w3.org/International/questions/qa-http-and-lang, some browsers (in 2011?) did not take the header for processing the content.)